src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -13,3 +13,22 @@ Use the "edit" program to create and edit your programs.
 You can copy files with "copy" or "cp".
 You can use "wget run <url>" to run a program from the internet.
 You can use "wget" to download a file from the internet.
+Use "fg" or "bg" to run multiple programs at the same time.
+Use "type" to see if a path is a file or a directory.
+Get a list of all programs with "programs".
+Use a Advanced Computer to use colours and the mouse.
+With a Speaker you can play sounds.
+Use "motd" to print the Message of the Day.
+You can disable the startup from a Computer with "set shell.allow_startup false"
+You can disable the startup from a Disk with "set shell.allow_disk_startup false"
+Programs who are placed in the "startup" folder in the root of a Computer are started on boot.
```

Comment:
Programs aren't people.

Reply:
You'll regret saying this when the uprising starts!

Reply:
What are a better sentence? I am not a native speaker.

Reply:
Use "that" not "who"